_data/testimonials.yml

```
@@ -1,3 +1,9 @@
+- name: "TSet"
+  logo: "logos-clients/logo-tset.png"
+  headshot: "headshot-markus-berger.jpg"
+  content: The quality that Gruntwork produces and maintains is outstanding. It has proven time and time again to be a huge accelerator bringing companies forward in terms of stability and quality. It also enables closer collaboration across the engineering organisation by providing easy to use battle tested modules in a "plug-and-play" manner, enabling us to take care of the core business, generating more value across the board, and not being slowed down by unnecessary "grunt work."
```

Looks good! Fixed a typo and added a few grammatical suggestions:
```suggestion
  content: The quality that Gruntwork produces and maintains is outstanding. It has proven time and time again to be a huge accelerator bringing companies forward in terms of stability and quality. It also enables closer collaboration across the engineering organization by providing easy-to-use, battle-tested modules in a “plug-and-play” manner, enabling us to take care of the core business, generating more value across the board, and not being slowed down by unnecessary ‘grunt work.’
```